x-pack/plugins/security_solution/cypress/integration/detection_alerts/alerts_details.spec.ts

```diff
@@ -80,4 +82,12 @@ describe('Alert details with unmapped fields', () => {
         expect($tableContainer[0].scrollLeft).to.equal(0);
       });
   });
+
+  it('Opens a new timeline investigation', () => {
```

Comment:
Can you test that the number in the flyout matches the number that loads in the timeline?

Reply:
Sure -> https://github.com/elastic/kibana/pull/131255/commits/983d852a3463597e63dc7e2093740d1c3ca5e9e3